### DISCUSSION TITLE
FN5 MyOpenMath

### DISCUSSION THREAD

**Discussion by AbbyANoble** [TeamBasedInquiryLearning/precalculus#335] [General]
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
I noticed that problem 3 asks students to find the domain of each combined function. Although domain is covered in a previous section, did we want to include it here? The activity book does not ask students to find the domain of any of the combined functions. Do we want to add that in the activity book?

**Comment by tdegeorge** [comment]
We also don't cover a lot of finding the domain of a composite function. We do have one part where students are asked to find the domain and range when given tables, but not from a composed function given equations. Like this (question 12):
<img width="530" alt="Screenshot 2024-07-04 at 9 52 23 AM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/7dfd7583-fbbf-461a-99aa-8ccd668c56cc">

**Comment by tdegeorge** [comment]
So...this may not apply here, but did we put decomposing functions somewhere?
